src/LinAlg/matrix.h

```diff
@@ -1360,6 +1408,24 @@ namespace utl //! General utility classes and functions.
   }
 
   template<class T> inline
+  bool matrix<T>::multiply(const std::vector<T>& X, std::vector<T>& Y,
+                           double alpha, double beta, bool transA,
+                           int stridex, int stridey,
+                           int ofsx, int ofsy) const
+  {
+    if (!this->compatible(X,transA)) return false;
+
+    for (size_t i = ofsy; i < Y.size(); i += stridey)
+      for (size_t j = ofsx; j < X.size(); j += stridex)
+        if (transA)
+          Y[i] += beta * Y[i] + alpha * THIS(j+1,i+1) * X[j];
+        else
+          Y[i] += beta * Y[i] + alpha * THIS(i+1,j+1) * X[j];
+
```

Comment:
Dette kan da ikke bli riktig, beta mulitpliseres inn mange ganger for hver j? Du må vel her heller gjøre denne multipliseringen (når beta er forkjellig fra 1) forkant i en separat løkke, før hovedløkka. eller blanke ut Y hvis beta = 0.

Når er det vel skjelden at det kjøres uten BLAS, men likevel...